src/Goerli-DssSpell.sol

```diff
@@ -24,14 +25,37 @@ import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralOnboardingAction } from "./Goerli-DssSpellCollateralOnboarding.sol";
 
+interface VatLike {
+    function Line() external view returns (uint256);
+    function file(bytes32, uint256) external;
+    function ilks(bytes32) external returns (uint256 Art, uint256 rate, uint256 spot, uint256 line, uint256 dust);
+}
+
+interface StarknetLike {
+    function setCeiling(uint256) external;
+}
+
 contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
 
     // Provides a descriptive tag for bot consumption
     string public constant override description = "Goerli Spell";
 
+    address constant STARKNET_ESCROW_MOM    = 0x464379BD1aC523DdA45b7B78eCB1F703661cad2a;
+    address constant STARKNET_ESCROW        = 0x38c3DDF1eF3e045abDDEb94f4e7a1a0d5440EB44;
+    address constant STARKNET_DAI_BRIDGE    = 0xd8beAa22894Cd33F24075459cFba287a10a104E4;
```

Comment:
When checking wards, it was clear this was deployed before the ESM changes.  Indeed the old ESM is made a ward on this contract, but not the new one.  We should check this in `mainnet` too.  I think we need to `deny` the old ESM ward in this spell, and add the new ESM too.

Reply:
Great catch. Pushed an update to deauth the old ESM and auth the new one.